types/store_internal_test.go

```diff
@@ -0,0 +1,58 @@
+package types
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
i couldnt run the tests as this caused an import cycle
```suggestion
```

i wasnt able to run the tests due of import cycle here